spec/features/pom_staff_female_feature_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+feature "get poms list" do
+  before do
+    signin_spo_user
+
+    # mock to ensure we are in a women's prison
+    allow(PrisonService).to receive(:womens_prison?).and_return(true)
+    visit prison_poms_path('LEI')
+  end
+
```

Comment:
This feels wrong to me - ideally we should not be using VCR for new tests. Our pattern is to use WebMock stubs and build(:womens_prison) to make a Women's prison.